tensorflow/compiler/xla/service/hlo_instruction.cc

```diff
@@ -481,6 +481,8 @@ StatusOr<std::unique_ptr<HloInstruction>> HloInstruction::CreateFromProto(
       static_cast<HloCustomCallInstruction*>(instruction.get())
           ->set_batch_group_count(
               std::max(static_cast<int64>(proto.batch_group_count()), 1LL));
+      static_cast<HloCustomCallInstruction*>(instruction.get())
```

Comment:
@jlebar given all the other casts here are `static_cast` rather than `Cast`, should this one change?

Reply:
That repeated casting there is kind of gross.  It sort of looks like the author was not aware that you can declare variables inside of a switch statement if you brace the `case`?

So personally I'd declare a variable and do the cast just once -- then you can do a proper `Cast` call.  But if you don't want to do that, then yeah, doing `static_cast` here seems like a good idea.